internal/xds/rbac/rbac_engine.go

```diff
@@ -131,9 +139,37 @@ func newEngine(config *v3rbacpb.RBAC) (*engine, error) {
 		}
 		policies[name] = matcher
 	}
+
+	auditOptions := config.GetAuditLoggingOptions()
+	auditLoggers := []*audit.Logger{}
```

Comment:
If taken the suggestion below, this can be defined later and the early return will assign that field to nil.

Also `var auditLoggers []*audit.Logger` is slightly preferred over `auditLoggers :=` for declaring slices, from my experience.